docs/_includes/options/events/jquery.html

```diff
@@ -43,8 +43,17 @@ <h2 id="events-public">
   <h3>
     Does Select2 include extra information in these events?
   </h3>
+If your server part returns additional data in the json dictionary, you can access like this:
 
-  {% include options/not-written.html %}
+{% highlight js linenos %}
+$(function() {
+    $('#mywidget').on('select2:select', function(evt){
+        alert(evt.params.data.mydata);
```

Comment:
Ideally this wouldn't necessarily be documenting the fact that you can send back additional data on the selected data object, but instead it would be documenting the fact that `evt.params.data` exists on `select2:select`. Select2 passes a bunch of data along with events through `evt.params`, and the end goal for this section was to document that.